packages/appstore/src/components/wallets-banner/wallets-banner-ready.tsx

```diff
@@ -4,14 +4,14 @@ import { Button, Icon, Text } from '@deriv/components';
 import { Localize, localize } from '@deriv/translations';
 import { isMobile } from '@deriv/shared';
 import { TWalletsImagesListKeys } from 'Assets/svgs/wallets/image-types';
-import { useStore } from '@deriv/stores';
+import { observer, useStore } from '@deriv/stores';
 
 // just for now for testing purpose, in the future 'is_eu' value will be taken from the store
 type TWalletsBannerReady = {
     is_eu?: boolean;
 };
 
-const WalletsBannerReady = ({ is_eu }: TWalletsBannerReady) => {
+const WalletsBannerReady = observer(({ is_eu }: TWalletsBannerReady) => {
```

Comment:
I didn't use observer here because logout function never changes and we don't need to check it and repaint component, so in my opinion using observer is redundantly. But, as I said before, you can ignore it

Reply:
From my understanding if you use `useStore` you need to have `observer` and it was raised by @farzin-deriv aswell regarding the use of `observer` in the WalletsBanner file from a previous commit. Let's have more people review and give their opinions. 😃 👍🏼

Reply:
@aizad-deriv In my opinion we need `observer` to check store parameter changes, and if parameter (here is `logout`) changes then it calls repaint of component. Anyway it's out of the scope of this PR, thank you for your response. I'll approve this PR when you fix CircleCI issue :)

Reply:
@sergei-deriv Yes you are correct, The problem is since the `useStore` hook is already there later in the future someone will come here and use an observable value thinking that it will work. 

Unfortionally if you miss `observer` there won't be any error or any way to know it's been missed and the process of debugging why the state is not changing is a pain (been there :D). 

So we are advising wrapping the component with `observer` whenever we are using the `useStore` hook to avoid this kind of situation, Since `observer` internally uses `React.memo` there won't be any rerendering happening of there is no observable value in use.

Reply:
@farzin-deriv Thanks a lot, great answer as always!!!